mlrun/artifacts/base.py

```diff
@@ -138,6 +141,8 @@ def __init__(
         self.db_key = db_key
         self.extra_data = extra_data or {}
         self.unpackaging_instructions = unpackaging_instructions
+        self.parent_uri = parent_uri
+        self.have_children = None
```

Comment:
What does `None` mean in this context? Is it the same as `False`? Then why not use `False`? Adding the 3rd option of using `None` to a boolean is logically confusing.